tests/python/utils.py

```diff
@@ -64,9 +65,8 @@ def get_test_prefix() -> str:
 
 
 def skip_aws_tests() -> bool:
-    dolma_tests_skip = os.environ.get(DOLMA_TESTS_SKIP_AWS_ENV_VAR)
-    LOGGER.info(f"{DOLMA_TESTS_SKIP_AWS_ENV_VAR}: {dolma_tests_skip}")
-    return (dolma_tests_skip or "false").lower() == "true"
+    dolma_tests_skip = yaml.safe_load(os.environ.get(DOLMA_TESTS_SKIP_AWS_ENV_VAR) or "false")
```

Comment:
safe_load is duplicative if casting to bool() regardless below right? More a nit than anything, I guess it wouldn't hurt...